amundsen_application/api/metadata/v0.py

```diff
@@ -494,3 +472,68 @@ def _log_get_user(*, user_id: str) -> None:
         logging.exception(message)
         payload = jsonify({'msg': message})
         return make_response(payload, HTTPStatus.INTERNAL_SERVER_ERROR)
+
+
+@metadata_blueprint.route('/user/bookmark', methods=['GET'])
+def get_bookmark() -> Response:
+    try:
+        user_id = request.args.get('user_id')
+        if user_id is None:
+            if app.config['AUTH_USER_METHOD']:
```

Comment:
does this part of auth work with flaskoidc? cc @verdan

Reply:
This should be fine. I haven't configured the auth_user_method yet, but I am pretty sure that I would be able to get this info from from flask's app context. thanks for confirming 👍